build.sbt

```diff
@@ -181,11 +181,51 @@ addCommandAlias(
   "+publishLocal; metals/runMain scala.meta.metals.DownloadDependencies "
 )
 
-commands += Command.command("save-expect") { s =>
-  "unit/test:runMain tests.SaveExpect" ::
-    s
+def configureMtagsScalaVersionDynamically(
```

Comment:
Wouldn't this be easier with the env var ?

Reply:
This function is anyway will be added in #3281 so I think it's better to use it as command.